app/src/main/java/com/example/whatsappclone/ui/MainScreen.kt

```diff
@@ -0,0 +1,101 @@
+package com.example.whatsappclone.ui
+
+import android.animation.Animator
+import android.animation.ObjectAnimator
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.viewpager.widget.ViewPager
+import com.example.whatsappclone.R
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainScreen : AppCompatActivity() {
```

Comment:
Always the activity name should end as an Activity, so for here it should have been MainActivity instead of MainScreen